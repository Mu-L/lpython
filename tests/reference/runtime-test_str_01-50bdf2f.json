{
    "basename": "runtime-test_str_01-50bdf2f",
    "cmd": "lpython {infile}",
    "infile": "tests/runtime_errors/test_str_01.py",
    "infile_hash": "7fb9748271af03bfa35e190a4514480662f42263ebfa0a6b2e082868",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "runtime-test_str_01-50bdf2f.stdout",
    "stdout_hash": "a618f191c5d794e3f906cbd2289fe3059057b7468a5f91f98fdc2bee",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 1
}