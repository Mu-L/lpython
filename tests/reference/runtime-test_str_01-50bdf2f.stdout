%string_descriptor = type { i8*, i64, i64 }
  %casted_string_ptr_to_desc = alloca %string_descriptor, align 8
%string_descriptor*

0 0
String index: -4is out of Bounds
